dab/app_telemetry.py

```diff
@@ -0,0 +1,14 @@
+from schema import dab_response_validator
+from time import sleep
+from DabTester import Default_Validations
+
+
+def start(test_result, durationInMs=0,expectedLatencyMs=0):
+    dab_response_validator.validate_start_device_telemetry_response_schema(test_result.response)
+    sleep(5)
```

Comment:
Can you recheck these sleep numbers and minimize them? Outside of app start/stop I dont think we need to wait for 5 seconds in general.

Reply:
SG